packager_test.go

```diff
@@ -75,7 +75,10 @@ func TestSameDomain(t *testing.T) {
 	defer server.Close()
 
 	pkg := webpackager.NewPackager(makeConfig(server))
-	pkg.Run(urlutil.MustParse("https://example.org/hello.html"), date)
+	err := pkg.Run(urlutil.MustParse("https://example.org/hello.html"), date)
```

Comment:
Good catch. pkg.Run() had initially no return value (aka. "void") but changed to returning error later to make it server-mode friendly.